app/cdap/services/PipelineUtils/index.ts

```diff
@@ -42,6 +44,17 @@ export function duplicatePipeline(pipelineName: string, config?: IPipelineConfig
   });
 }
 
```

Comment:
add jsdoc to this

Reply:
I would like for all exported or complicated functions to have js doc if possible

Reply:
all _new_ exported or complicated functions - no need to add to old ones unless you heavily alter them